fix(ChatbotConversationHistoryNav): Add additional menu item props (#374)
patternfly/chatbot: packages/module/patternfly-docs/content/extensions/chatbot/examples/UI/UI.md

```

### Drawer with active conversation

> If you're showing a conversation that is already active, you can set the `isActive` prop on your `Conversation` to true for a visual state change.

Reviewer: ```suggestion
If you're showing a conversation that is already active, you can set the `isActive` prop on your `Conversation` to `true` to apply an active visual state.
```

> If you're showing a conversation that is already active, you can set the `activeItemId` prop on your `ChatbotConversationHistoryNav` to apply an active visual state.

Reviewer: ```suggestion
If you're showing a conversation that is already active, you can set the `activeItemId` prop on your `<ChatbotConversationHistoryNav>` to apply an active visual state.
```

Author: Done! Thank you.
